Dockerfile

```diff
@@ -15,12 +15,11 @@ ENV PYTHONUNBUFFERED=1
 
 COPY --from=builder /app .
 
+# Install dependencies
 RUN pip install --upgrade pip && \
     pip install -r requirements.txt
```

Comment:
Ensure that the `requirements.txt` file is present in the context of the Docker build. If it's missing, the `pip install -r requirements.txt` command will fail.